mayastor/Cargo.toml

```diff
@@ -114,5 +114,5 @@ version = "0.8.2"
 [dev-dependencies]
 assert_matches = "1.5.0"
 composer = { path = "../composer" }
-nvmeadm = { path = "../nvmeadm" }
+nvmeadm = { path = "../dependencies/nvmeadm" }
```

Comment:
Do we still have dependency on nvmeadm after using `libnvme-rs` crate

Reply:
As per my comment I need to rebase and update the PR

Reply:
@akhilerm I've updated the PR